scripts/linters/custom_eslint_checks/rules/e2e-browser-nested-awaits.spec.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
Could you make these tests more comprehensive? Right now they don't really cover all the edge cases. These tests are fast to run, so it's fine to have a lot of them

Reply:
I have added additional test cases, let me know I should add more.